script.js

```diff
@@ -0,0 +1,110 @@
+const foodLists = [
+  {
+    image: 'images/plate1.png',
+    name: 'Burger Rafiné',
+    prix: '4.5 USD',
+    description:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque dignissim eu ante vitae imperdiet.',
+  },
+  {
+    image: 'images/plate2.png',
+    name: 'Cheeps special',
+    prix: '6 USD',
+    description:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer condimentum congue aliquam.',
+  },
+  {
+    image: 'images/plate3.png',
+    name: 'Pizza Zairois',
+    prix: '8.8 USD',
+    description:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam laoreet accumsan turpis eget lacinia.',
+  },
+  {
+    image: 'images/plate1.png',
+    name: 'Salad Avocat',
+    prix: '12 USD',
+    description:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed id vulputate magna.',
+  },
+  {
+    image: 'images/plate1.png',
+    name: 'Smoothy',
+    prix: '17 USD',
+    description:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed id vulputate magna.',
+  },
+  {
+    image: 'images/plate1.png',
+    name: 'Rapid coffee',
+    prix: '11.5 USD',
+    description:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed id vulputate magna.',
+  },
+];
+
+window.onload = () => {
+  foodLists.forEach((foodList) => {
+    document.querySelector('.deliveryCard').innerHTML += `
+          <div class="card2">
+              <img class="lecturerPic" src="${foodList.image}" alt="${foodList.name}">
+              <div class="lecturer-info">
+              <h3 class="lecturer-name">${foodList.name}</h3>
+              <h4 class="lecturer-position">${foodList.prix}</h4>
+              <div class="grey-line"></div>
+              <p class="lecturer-detail">${foodList.description}</p>
+              </div>
+          </div>`;
+  });
+  document.querySelector('.foods').innerHTML = `
+    <div class="card2">
+        <img class="lecturerPic" src="${foodLists[0].image}" alt="${foodLists[0].name}">
+        <div class="lecturer-info">
+        <h3 class="lecturer-name">${foodLists[0].name}</h3>
+        <h4 class="lecturer-position">${foodLists[0].prix}</h4>
+        <div class="grey-line"></div>
+        <p class="lecturer-detail">${foodLists[0].description}</p>
+        </div>
+    </div>
+    <div class="card2">
+        <img class="lecturerPic" src="${foodLists[1].image}" alt="${foodLists[1].name}">
+        <div class="lecturer-info">
+        <h3 class="lecturer-name">${foodLists[1].name}</h3>
+        <h4 class="lecturer-position">${foodLists[1].prix}</h4>
+        <div class="grey-line"></div>
+        <p class="lecturer-detail">${foodLists[1].description}</p>
+        </div>
+    </div>`;
+};
+
+const menu = document.querySelector('.mobile-menu');
```

Comment:
- [ ] You navbar menu working good in `Home` page but unfortunately it doesn't work on `About` page, Kindly fix this issue 

![burger](https://user-images.githubusercontent.com/80895497/153725079-53902b44-917a-4a54-92bd-671444ef5b71.PNG)